### PR TITLE
Feature modal

### PR DESCRIPTION
**Pull request by shadabwahidullah**
I have worked on the modal I have set the display to be shown by default but the way it works is when you click on a comment button it will popup.

### CODE REVIEW

**Review by jonathastavares — CHANGES_REQUESTED** [review]
👍 You have done a great job so far

Great use of Async() function in the createModal method!! Good job so far!! 

♻️ There are some points that I think, It would be better to work on.

File: src/modal.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    modalTitle,
+    modalGenre,
+    modalRating,
+    modalWebsite,
+    modalDesc,
+  );
+}
+
+// this function is called when you click on comment button
+createModal(3498);
```
Comment (jonathastavares) [inline src/modal.js:63]: Now you should add event listeners for every comment button, create an unique Id for it in index.html and check use querySelectorAll to get all of them, and add a forEach() loop to check for every button.

File: src/style.css [hunk 2]
```diff
@@ -15,6 +15,16 @@ body {
   font-weight: bold;
 }
 
+#scrollableModal {
+  background: rgba(255, 228, 196, 0.8);
+  display: block;
```
Comment (jonathastavares) [inline src/style.css:49]: Display property is block by default, why did you use that here? Was it necessary?

**Review by jonathastavares — CHANGES_REQUESTED** [review]
👍 You have done a great job so far

Great use of Async() function in the createModal method!! Good job so far!!

♻️ There are some points that I think, It would be better to work on.

File: src/modal.js (hunk 1 above)
Comment (shadabwahidullah) [reply src/modal.js:63]: I am waiting for you to complete the changes then we can merge your code to the development branch. I like to add the listeners where you populate your games.

File: src/style.css (hunk 2 above)
Comment (shadabwahidullah) [reply src/style.css:49]: It's just to show the popup when the page loads, so you could see it. I will remove it along the way. No, It's not necessary.

**Review by jonathastavares — APPROVED** [review]

**Comment by jonathastavares** [comment]
Status: Approved!

There's nothing else to say than... It's time to merge!

Congratulations finishing this milestone. Happy Coding!

**Pull request merged**